apps/datahub/src/app/record/record-otherlinks/record-otherlinks.component.html

```diff
@@ -11,9 +11,20 @@
   ></gn-ui-previous-next-buttons>
 </div>
 <ng-container *ngrxLet="otherLinks$ as otherLinks">
+  <gn-ui-carousel
+    *ngIf="isMobile$ | async"
+    containerClass="gap-4 pt-[26px] pb-[38px]"
+    (currentStepChange)="updateView()"
+  >
+    <gn-ui-link-card
+      *ngFor="let otherLink of otherLinks; let first = first; let last = last"
```

Comment:
`let first = first; let last = last`
are they used here? (looks like they are not)